SettingsFile/SettingsFile/JsonSettings.cs

```diff
@@ -120,5 +136,17 @@ public JsonSettings ReopenFile()
     /// Saves the data in this instance to file.
     /// </summary>
     public void Save()
-        => File.WriteAllText(SettingsFile.SettingsPath, Serialize(this));
+    {
+        // trap default instance values and return.
+        if (string.IsNullOrEmpty(this.ElsDir)
+            && this.Sources.Equals(Array.Empty<string>())
+            && this.WindowIcon.Equals(-1)
+            && this.IconWhileElsNotRunning.Equals(-1)
+            && this.IconWhileElsRunning.Equals(-1))

```

Comment:
If only I could simplify this to ``if (this is default)`` and have it basically mean the same thing. 😭